packages/sites/test/pages/update-page.test.ts

```diff
@@ -92,6 +92,45 @@ describe("updatePage", () => {
     );
   });
 
+  it("handles old call signature", async () => {
```

Comment:
again, this looks to me like the _new_ signature b/c of `requestOptions.patchList`. if so this would be the test we'd want to keep. Is there another test for the deprecated signature (model, patchList, options)? If so _that_ one needs to be marked as such w/ explicit comments like `// this should be dropped when we bump major`